packages/preview/bookletic/0.1.0/src/lib.typ

```diff
@@ -0,0 +1,132 @@
+// This function creates a signature (booklet) layout
+// It takes various parameters to configure the layout
+#let sig(
+  signature_paper: "us-letter", // Paper size for the booklet
+  page_margin_top: 0.25in, // Top margin for each page
+  page_margin_bottom: 0.25in, // Bottom margin for each page
+  page_margin_binding: 0.25in, // Binding margin for each page
+  page_margin_edge: 0.25in, // Edge margin for each page
+  page_border: luma(0), // Whether to draw a border around each page
+  draft: false, // Whether to output draft or final layout
+  pNum_pattern: "1", // Pattern for page numbering
+  // TODO: add ability to set a shape behind page number
+  pNum_placment: top, // Placement of page numbers (top or bottom)
+  pNum_align_horizontal: center, // Horizontal alignment of page numbers
+  pNum_align_vertical: horizon, // Vertical alignment of page numbers
+  pNum_size: 12pt, // Size of page numbers
+  pNum_pad_horizontal: 1pt, // Horizontal padding for page numbers
+  pNum_border: luma(0), // Border color for page numbers
+  pad_content: 5pt, // Padding around page content
```

Comment:
It is considered more idiomatic to use kebab-case names and not snake_case or camelCase ones. This is not a hard requirement however, and your package can be accepted as is if you want.